src/js/components/pages/scene-media-browser.jsx

```diff
@@ -101,9 +107,11 @@ var SceneMediaBrowser = React.createClass({
     getLastSplit: function() {
         if (localStorage.getItem("splitPos") === null) {
             //default split
-            return 150;
-        } else { 
-            return parseInt(localStorage.getItem('splitPos'), 10)
+            return DEFAULT_SPLIT_PANE_POS;
+        } else {
+            // APEP if we've cached a value larger than the dimensions of the component, lets fallback to default
+            var cachedSplitPos = parseInt(localStorage.getItem('splitPos'), 10);
```

Comment:
Given this was added before I added the maxSize to the <SplitPane> might not be necessary anymore.